🀄️[zh_cn]Simplified Chinese translation repair

Catchmons opened the pull request:
又是实机找错的一天，还发现了些英文原文的错误，稍后另起一个 PR。部分修改内容出自个人意愿，欢迎各位讨论。

It's another day of finding in-game problem, and I also discovered some mistake in the English original text. I will start another PR later. Some changes were made at the discretion of individuals, and i welcome everyone's discussion.

Catchmons committed:
一些修改

Catchmons commented:
本次修改省流：

- 依据百度百科修正部分反应的解释
- 添加“水合”条目（个人认为水解并不能完全替代水合，尤其是乙烯水化法制醇）
- 调整部分语序
- 修复错误（如三硝基甲苯合成页处显示错误）
- 反应的解释部分加上句号（这个不太确定合不合适）

Catchmons commented:
@helloworld069

Catchmons committed:
just tnt

Catchmons committed:
this should be in another pr

BytinYarota commented:
> * 添加“水合”条目（个人认为水解并不能完全替代水合，尤其是乙烯水化法制醇）

这个当时翻译的时候是本着完全忠于原文进行的，像水合水解之类的确实就没法翻译畅通，所以我个人同意这一修改
> * 反应的解释部分加上句号（这个不太确定合不合适）

我建议去掉句号，参考思索等诠释性的文本一般都没有句号，尽管英语原文存在。

u1s1这些汉语翻译问题还是在fork或者mcmod里讨论吧，直接在原仓库里讨论有点占用公共资源我觉得(
@Catchmons

BytinYarota commented:
以及这种小修也许攒多一点再发pr会好一些？个人感觉时不时发一个可能会比较打搅人(

Catchmons commented:
> 以及这种小修也许攒多一点再发pr会好一些？个人感觉时不时发一个可能会比较打搅人(

主要是这些东西一次找不完，这样吧之后的版本更新后找到错就发个pr然后标注成draft？百科的论坛都没啥人用还是算了吧（

Catchmons committed:
删去句号

Catchmons committed:
Update zh_cn.json

Catchmons committed:
Update zh_cn.json

Catchmons committed:
调整级的定义

详见：https://baike.baidu.com/item/%E7%B3%BB%E7%BB%9F%E5%91%BD%E5%90%8D%E6%B3%95?fromModule=lemma_search-box

Catchmons committed:
一些调整

对部分反应的名称和描述进行了一些修改

Catchmons committed:
小修改

Catchmons committed:
小修改

Catchmons committed:
Merge branch 'petrolpark:1.20.1' into 1.20.1
